The simplify linter wants you to change a line:
```
pkg/chartutil/create.go:252:2:warning: 'if err != nil { return cdir, err }; return cdir, nil' can be simplified to 'return cdir, err' (gosimple)
```

I'll add a comment on the code.

--- a/pkg/chartutil/create.go
+++ b/pkg/chartutil/create.go
@@ -146,8 +146,8 @@ const defaultNotes = `1. Get the application URL by running these commands:
   export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
   echo http://$NODE_IP:$NODE_PORT/login
 {{- else if contains "LoadBalancer" .Values.service.type }}
-**** NOTE: It may take a few minutes for the LoadBalancer IP to be available.                      ****
-****       You can watch the status of by running 'kubectl get svc -w {{ template "fullname" . }}' ****
+     NOTE: It may take a few minutes for the LoadBalancer IP to be available.
+           You can watch the status of by running 'kubectl get svc -w {{ template "fullname" . }}'
   export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
   echo http://$SERVICE_IP:{{ .Values.Master.ServicePort }}
 {{- else if contains "ClusterIP"  .Values.service.type }}

--- a/pkg/chartutil/create.go
+++ b/pkg/chartutil/create.go
@@ -249,8 +249,5 @@ func Create(chartfile *chart.Metadata, dir string) (string, error) {
 
 	// Write out _helpers.tpl
 	val = []byte(defaultHelpers)
-	if err := ioutil.WriteFile(filepath.Join(cdir, TemplatesDir, HelpersName), val, 0644); err != nil {
-		return cdir, err
-	}
-	return cdir, nil
+	return cdir, ioutil.WriteFile(filepath.Join(cdir, TemplatesDir, HelpersName), val, 0644)
 }

